.github/workflows/pypi-publish.yaml

```diff
@@ -23,18 +22,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install uv
-        uses: astral-sh/setup-uv@v3
-        with:
-          enable-cache: true
-          cache-dependency-glob: pyproject.toml
-
-      - name: Setup Python
-        uses: actions/setup-python@v5
-        with:
-          python-version-file: pyproject.toml
-
-      - name: Install Python dependencies
-        run: uv pip install .
```

Comment:
None of this stuff is actually necessary to build the package, so we can remove it.